packages/api/tests/lambdas/test-message-consumer.js

```diff
@@ -110,10 +120,13 @@ test('handler processes records as expected', async (t) => {
   );
   t.true(fetchEnabledRulesStub.calledOnce);
 
-  t.true(publishMessageStub.withArgs({
+  const expectedArgs = {
     TopicArn: process.env.FallbackTopicArn,
     Message: JSON.stringify(erroringMessage),
-  }).calledOnce);
+  };
+  const publishCalls = snsMock.commandCalls(PublishCommand);
+  t.true(publishCalls.length > 0);
```

Comment:
Another `calledOnce` vs `> 0`.

Reply:
https://github.com/nasa/cumulus/pull/3606/commits/881e7bf190890a675b18f4d947f9d10ed1abc140

I don't think there's a good way to use the sinon `calledOnce` helper unfortunately. The `commandCalls` function returns a list of calls. Not seeing a great way to get the spy directly.